### DISCUSSION TITLE
Is it possible to run on a PC with AMD Ryzen 5 5500U with Linux thought a OSX-KVM Virtual Machine?

### DISCUSSION THREAD

**Discussion by paolostefanucci** [Chapoly1305/FindMy#12] [Q&A]
Hello @Chapoly1305 I wanted to ask if the request_reports.py and other python scripts is it possible to run on a PC with AMD Ryzen 5 5500U with Linux thought a KVM Virtual Machine with Monterey o Ventura instead of use Docker and anisette-v3-server

I mean  this:  https://github.com/kholia/OSX-KVM

**Comment by Chapoly1305** [comment]
Hello, 
Thank you for the question. I have a peek on the project you provided, it seems to be a project that we have not considered previously. 

The [OpenHayStack](https://github.com/seemoo-lab/openhaystack) supports calling the Apple API by using a plug-in for Mail, no need to install or use the Docker or anisette server. The OpenHayStack can only run between MacOS 10~11, and it will not work on Mac 12 and future due to Apple removed the support of Mail plug-in. I have personally tested with MacOS 10 no long ago, it works as expected. 

In my personal consideration, using this OSX-KVM might be overkill for most of the users. If you don't have an environment yet and wish to give a try, I appreciate your sharing here. If you do own an environment, I recommend OpenHayStack and give it a try. I do not have the knowledge for other methods at this moment. 

Best.

**Reply by paolostefanucci** [reply]
I want report to you that I was successful able to run OpenHayStack on OSX-KVM  with BigSur on PC notebook with Ryzen 5500U and Linux.
But I can not see the map, and it is slow, maybe because my integrated GPU is non an Hackintosh compatible graphic card so I don't have a graphic accelerator, I think that this work in framebuffer. In any case, I'm happy with the result

**Reply by Chapoly1305** [reply]
Congrats! Glad to hear it actually worked!
